athena/main.py

```diff
@@ -47,7 +47,9 @@
     "stargan" : StarganModel,
     "tts_transformer": TTSTransformer,
     "fastspeech": FastSpeech,
-    "speaker_resnet": SpeakerResnet
+    "speaker_resnet": SpeakerResnet,
+    "gender_resnet": SpeakerResnet,
```

Comment:
We should probably change the name of SpeakerResnet to something like ClassificationResnet and use the keyword "task" to distinguish between each task